File: dongyimai-sellergoods-interface/src/main/java/com/offcn/sellergoods/service/BrandService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbBrand;

import java.util.List;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult findPage(int pageNum,int pageSize);

    public void addBrand(TbBrand brand);

    public TbBrand findOne(Long id);

    public void update(TbBrand brand);

    public void delete(Long[] ids);

    public PageResult findPage(int pageNum,int pageSize,TbBrand brand);


}

File: dongyimai-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {//一个brandService的类

    this.findAll=function () {
       return  $http.get('../brand/findAll.do');
    }

    this.add=function (entity) {
        return  $http.post('../brand/addBrand.do', entity);
    }

    this.update=function (entity) {
        return   $http.post('../brand/update.do', entity);
    }

    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }

    this.dele=function (selectIds) {
        return $http.get('../brand/delete.do?ids='+selectIds);
    }

    this.search=function (page, rows, searchEntity) {
        return $http.post('../brand/search.do?page='+page+"&rows="+rows, searchEntity);
    }
})